doc/Makefile

```diff
@@ -47,139 +41,185 @@ help:
 	@echo "  pseudoxml  to make pseudoxml-XML files for display purposes"
 	@echo "  linkcheck  to check all external links for integrity"
 	@echo "  doctest    to run all doctests embedded in the documentation (if enabled)"
+	@echo "  coverage   to run coverage check of the documentation (if enabled)"
+	@echo "  dummy      to check syntax errors of document sources"
 
+.PHONY: clean
 clean:
 	rm -rf $(BUILDDIR)/*
-	rm -rf generated/*
```

Comment:
I think this is still useful for clearing pages generated by sphinx for API docs.